### PR TITLE
Conventions for git commits and Pull Requests

### PR DESCRIPTION
**Pull request by zupo**
So far, we have not had a concrete definition of how our git commits and pull requests should look like. We worked off previous experience and conventions in the wider tech community. However, since we've grown considerably in the past months, there has been some opposing opinions. This PR attempts to resolve any remaining open issues as to how git commits and pull requests should look like in Niteo's repositories. 

The PR adds two sections to the Work Process document that explain and provide examples on good git commit messages and pull requests. It is expected that every Nitean follows these conventions.

I've intentionally left the `TODO`s where links to excellent commits/pull request should be to encourage everyone to create the best possible commits/pull requests in the next 10 days. We'll then choose the best 3 of each and the authors will enjoy eternal glory. 

I've also added a tiny bit of whitespace cleanup and some typo fixes.

Refs niteoweb/ebn#2280.

### CODE REVIEW

File: work-process.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(softwa
...
+## How to create a User Story
 
 - In Operations repository when you open an issue, it has a template that helps you define the User Story.
 - User Story in this step must be moved into `Backlog` pipeline in ZenHub.
 - Various labels are added (see Issue Labels section).
 - The Scrum Master and Product Owner verify if the User Story is well defined.
   * If it is not they assign people from whom they want to receive feedback for better definition of the User Story.
   * Once these people post their feedback, they unassign themselves from the User Story.
-- Once the Scrum Master and Product Owner agree that the User Stories well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this online poker planning begins.
+- Once the Scrum Master and Product Owner agree that the User Storie is well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this, online poker planning begins.
```
Comment (dz0ny) [inline work-process.md]: typo **Storie** :D should be Story

Comment (zupo) [reply work-process.md]: tnx

**Comment by dmurko** [comment]
I would put this at the bottom of the doc.

**Comment by cas--** [comment]
Additional User Story references:

Commit messages: https://github.com/niteoweb/operations/issues/273
Pull Requests: https://github.com/niteoweb/operations/issues/276

File: work-process.md [hunk 2]
```diff
@@ -63,6 +63,77 @@ Activities that need to be performed so this user story can be moved to `Done`.
...
+A few examples of excellent commits:
+ * TODO
+ * TODO
+ * TODO
+
+
+### Pull Request
+
+When your commit is ready to be applied to code/text inside GitHub, you need to present it with a Pull Request.
+Similarly to commits, a Pull Request should only include commits on a single topic. For example, if you are adding a new section to this Handbook, but then notice that some other section is also missing, that second section should be in a separate commit.
```
Comment (cas--) [inline work-process.md]: Is it meant to end with 'separate commit' or did you mean 'separate PR'? The surrounding text and paragraph below suggest the latter.

Comment (zupo) [reply work-process.md]: PR

**Comment by zupo** [comment]
@dmurko: I strongly disagree. The document in that places talks about how to write and work on user stories. This directly relates, as a lot of the work on user stories involves commits and pull requests.

**Pull request merged**